Update `Client/Menu tips` (#12538)
ppy/osu-wiki: wiki/Client/Menu_tips/en.md

  - hints
---

> <!-- For translators: to translate this page, select the corresponding language in stable and open `Localisation/XX.txt` in your osu! installation folder. Translation keys starting with MenuTip_ and EditorTip_ contain the respective translated tips. The lazer tips can be translated on Crowdin: https://crowdin.com/editor/osu-web/656 -->

Reviewer: i do agree with this, but it could imply moreso that the translations can be sourced from crowdin than that the reader should be taking part in translations there (to partly address clayton's review)

very much a nit yes

```suggestion
<!-- For translators: to translate this page, select the corresponding language in stable and open `Localisation/XX.txt` in your osu! installation folder. Translation keys starting with MenuTip_ and EditorTip_ contain the respective translated tips. The lazer tips are translated on Crowdin: https://crowdin.com/editor/osu-web/656 -->
```

> <!-- For translators: to translate this page, select the corresponding language in stable and open `Localisation/XX.txt` in your osu! installation folder. Translation keys starting with MenuTip_ and EditorTip_ contain the respective translated tips. The lazer tips can be translated on Crowdin: https://crowdin.com/editor/osu-web/656/en-de?view=comfortable&filter=basic&value=0 -->

Reviewer: ```suggestion
<!-- For translators: to translate this page, select the corresponding language in stable and open `Localisation/XX.txt` in your osu! installation folder. Translation keys starting with MenuTip_ and EditorTip_ contain the respective translated tips. The lazer tips can be translated on Crowdin: https://crowdin.com/editor/osu-web/656 -->
```
